__tests__/Formsy-spec.tsx

```diff
@@ -61,92 +63,63 @@ describe('Setting up a form', () => {
   });
 
   it('should allow for inputs being added dynamically', () => {
-    const inputs = [];
-    let forceUpdate = null;
     let model = null;
-    class TestForm extends React.Component {
-      componentWillMount() {
-        forceUpdate = this.forceUpdate.bind(this);
-      }
-      render() {
-        return <Formsy onSubmit={formModel => (model = formModel)}>{inputs}</Formsy>;
-      }
-    }
-    const form = mount(<TestForm />);
 
-    // Wait before adding the input
-    setTimeout(() => {
-      inputs.push(<TestInput name="test" value="" key={inputs.length} />);
+    const form = mount(<DynamicInputForm onSubmit={formModel => (model = formModel)} inputName="test" />);
+    form.find('button').simulate('click');
+    form.update();
 
-      forceUpdate(() => {
-        // Wait for next event loop, as that does the form
-        immediate(() => {
-          form.simulate('submit');
-          test.ok('test' in model);
```

Comment:
As prove of the comment, `test.ok` is not exists but it is never run so the test not explodes => pass 

🙁